string name                         # object's name
geometry_msgs/Pose pose             # new pose
--- 
